kogito-postgresql/README.md

```diff
@@ -53,33 +54,10 @@ For example:
 helm install --values sql-script.yaml process-postgresql-persistence-quarkus kogito-postgresql
 ```
 
-## Concurrency Issues
+## Readiness Check
 Since the Kogito application and PostgreSQL instance are 
 started at the same time, concurrency issues can occur when 
```

Comment:
maybe, `race condition issues` fits better this problem.

Reply:
I agree, I just made this change